there is some ugly build hack required to make it compile and work properly on
armv7l - so far i did not manage to get this properly into the make/cmake setup
and some magic steps have to be done by hand still as a result: i simply
started the build via "RACK_DIR=/compile/Rack make dist" and as soon as it
started compiling i added "-latomic" to the end of
dep/cmake-build/CMakeFiles/plugin.dir/link.txt - not a proper solution, but at
least a way to get it built for now :)

diff --git a/surge/CMakeLists.txt b/surge/CMakeLists.txt
index bedf4929..becabf1c 100644
--- a/surge/CMakeLists.txt
+++ b/surge/CMakeLists.txt
@@ -52,7 +52,7 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang|GNU")
   add_compile_options(
     -Wno-multichar
     # Targeting Windows with GCC/Clang is experimental
-    $<$<NOT:$<BOOL:${WIN32}>>:-Werror>
+    # $<$<NOT:$<BOOL:${WIN32}>>:-Werror>
 
     # PE/COFF doesn't support visibility
     $<$<NOT:$<BOOL:${WIN32}>>:-fvisibility=hidden>
diff --git a/RackSDK.cmake b/RackSDK.cmake
index 0abbf43..f6b78d4 100644
--- a/RackSDK.cmake
+++ b/RackSDK.cmake
@@ -70,7 +70,7 @@ set_target_properties(${RACK_PLUGIN_LIB} PROPERTIES PREFIX "")
 
 # Since the plugin's compiler could be a different version than Rack's compiler, link libstdc++ and libgcc statically to avoid ABI issues.
 add_link_options($<$<CXX_COMPILER_ID:GNU>:-static-libstdc++> $<$<PLATFORM_ID:Linux>:-static-libgcc>)
-add_compile_options($<IF:$<STREQUAL:${CMAKE_OSX_ARCHITECTURES},arm64>,-march=armv8-a+fp+simd,-march=nehalem>)
+add_compile_options(-march=armv7 -mtune=cortex-a9 -mfpu=neon -mno-unaligned-access -DSIMDE_ENABLE_NATIVE_ALIASES)
 
 add_library(RackSDK INTERFACE)
 target_include_directories(RackSDK INTERFACE ${RACK_SDK_DIR}/include ${RACK_SDK_DIR}/dep/include)
diff --git a/Makefile b/Makefile
index 831cbbd..3f24b18 100644
--- a/Makefile
+++ b/Makefile
@@ -37,7 +37,7 @@ $(cmake_rack_plugin): CMakeLists.txt
 	cmake --install $(CMAKE_BUILD)
 
 rack_plugin: $(cmake_rack_plugin)
-	cp -vf $(cmake_rack_plugin) .
+	cp -vf $(cmake_rack_plugin) plugin.so
 
 # Add files to the ZIP package when running `make dist`
 dist: rack_plugin build/surge-data res
